management-app/src/main/java/com/nearsoft/fgaribay/mgmt/ManagementController.java

```diff
@@ -34,8 +35,11 @@ public String greetingForm(Model model) {
     }
 
     @RequestMapping(value = "/create", method = RequestMethod.POST)
```

Comment:
Try replacing this mapping annotations with `@GetMapping` or `@PostMapping` as needed.